exercises/allergies/canonical-data.json

```diff
@@ -1,109 +1,149 @@
 {
-    "allergic_to": {
-        "description": [
-            "Given a number and a substance, indicate whether Tom is allergic ",
-            "to that substance.",
-            "Test cases for this method involve more than one assertion.",
-            "Each case in 'expected' specifies what the method should return for",
-            "the given substance."
-        ],
-        "cases": [
+  "exercise": "allergies",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "allergicTo",
+      "comments": [
```

Comment:
We can have comments in the top level, in a test group (cases), or inside a test, so that people can keep them where they fit better. It this case, It is in the test group because it pertains just that group of test.